crates/net/network-types/src/peers/mod.rs

```diff
@@ -1,5 +1,141 @@
+pub mod addr;
+pub mod cmd;
+pub mod config;
+pub mod handle;
+pub mod kind;
 pub mod reputation;
-pub use reputation::ReputationChangeWeights;
+pub mod state;
 
-pub mod config;
+pub use cmd::PeerCommand;
 pub use config::{ConnectionsConfig, PeersConfig};
+pub use reputation::ReputationChangeWeights;
+
+use reth_ethereum_forks::ForkId;
+use tracing::trace;
+
+use crate::{
+    is_banned_reputation, PeerAddr, PeerConnectionState, PeerKind, ReputationChangeOutcome,
+    DEFAULT_REPUTATION,
+};
+
+/// Tracks info about a single peer.
+#[derive(Debug, Clone)]
+pub struct Peer {
```

Comment:
why do we need to move this?

this is currently a private type

Reply:
can move this back to network

Reply:
it's a public type on main though

Reply:
ah no I can't move it back, because it's used in `PeersHandle` impl